packages/hasura-storage-js/src/utils/index.ts

```diff
@@ -1,6 +1,7 @@
 import { StorageImageTransformationParams } from './types'
 
 export * from './types'
+export * from './upload'
```

Comment:
Why are there changes to hasura-storage-api in this PR?

Reply:
`toIso88591` was not exposed for tests. I reverted.